src/styl/_helpers/_cursor.styl

```diff
@@ -0,0 +1,11 @@
+// Cursor
+//
+// Change cursor in differents kind of specific situations
+// - **.zoomIn**: Change cursor to a magnifying glass to indicate to the user a zoom + .
+//
+// Markup: helpers-cursor.twig
+//
+// Styleguide: Helpers.Cursor
+
+.zoomIn
```

Comment:
We have to respect our naming convention here. You named this file `_cursor`, which is good since this helper concerns the cursor BUT the clas name should be the same. So:
```
.cursor
    &-zoomIn
```
See the logic?

Reply:
all right 👍